xbmc/Application.cpp

```diff
@@ -1809,6 +1809,8 @@ bool CApplication::LoadUserWindows()
               id = atol(pType->FirstChild()->Value());
           }
           std::string visibleCondition;
+          for (TiXmlElement* node = pRootElement->FirstChildElement("visible"); node; node = node->NextSiblingElement("visible"))
+            g_SkinInfo->ResolveIncludes(pRootElement->FirstChildElement("visible"));
```

Comment:
This one is goofy, I'd rather not resolve the visible conditions here at all. I expected `CGUIWindow::Load(pRootElement)` to handle the window visibility for custom windows like it does regular windows, but I just could not get that to work. 

`CGUIIncludes::ResolveExpressions` is private and it didn't seem like a good idea to change that and add a `CSkinInfo::ResolveExpressions` to pass it through, not for this one goofy instance.

In all other instances `CGUIIncludes::ResolveIncludes` already expands expressions into the XML before `CGUIControlFactory::GetConditionalVisibility` is called so it would mostly be duplicated effort.